lib/cv_dv_utils/uvm/Files.f

```diff
@@ -0,0 +1,30 @@
+${CV_DV_UTILS_DIR}/uvm/memory_partition/memory_partitions_pkg.sv
```

Comment:
A couple of comments about this file:
1. It should have a license header (two-liner is sufficient).
2. It seems that the user us expected to define `CV_DV_UTILS_DIR` shell env variable.  Is that correct?  If so, that should be documented in a README.

Reply:
Hi,

License has been added on uvm/Files.f

Regarding CV_DV_UTILS_DIR, you are rigth. Actually the README indicated that the user must define CV_DV_UTILS variable. I changed it to CV_DV_UTILS_DIR.

Regards,
Ludovic

Reply:
Hi @MikeOpenHWGroup ,

Is it ok now?

Thanks,
Ludovic